### PR TITLE
python: try bumping to Python 3.11 again [CPP-882]

### PR DESCRIPTION
**Pull request by silverjam**
Try bumping to Python 3.11 one more time.

Related PR: https://github.com/swift-nav/swift-toolbox-ci/pull/9

Test build of installers here: https://github.com/swift-nav/swift-toolbox/releases/tag/v4.0.9-py311

Jira: https://swift-nav.atlassian.net/browse/CPP-882

### CODE REVIEW

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4571790868)

**Comment by silverjam** [comment]
Anecdotally it looks like there's a good perf. bump with Python 3.11 on Windows

image

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4577851878)

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4578504777)

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4579018610)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -203,10 +203,10 @@ jobs:
         with:
           owner: swift-nav
           repo: swift-toolbox-ci
-          ref: main
+          ref: silverjam/py311
```
Comment (adrian-kong) [inline .github/workflows/main.yml:206]: dont forget to revert this after!!

**Review by adrian-kong — APPROVED** [review]
this is so good
howd u manage to fix the `import capnp` error i was having with #913

File: .github/workflows/main.yml (hunk 1 above)
Comment (silverjam) [reply .github/workflows/main.yml:206]: Thanks for the reminder

**Comment by silverjam** [comment]
> this is so good
> 
> howd u manage to fix the `import capnp` error i was having with #913 
> 

Not sure, was expecting to see that issue but it didn't pop up, so maybe it was a capnp bug for Python 3.11 that was fixed recently?

> tbh think it might be easier to remove py311 suffix so if it ever needs to change in future wont need digging everywhere but it this is very epic

Yeah, I was thinking it'd be a lot easier to update in the future if we drop the version from the folder name.  Will save that for another PR so I don't break this one.

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4591586234)

**Pull request merged**